src/Components/InventoryItemDetails/InventoryItemDetails.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import "./InventoryItemDetails.scss";
+import editLogo from '../../Assets/Icons/edit-24px-white.svg';
+import axios from 'axios';
+
+const InventoryItemDetails = () => {
+
+    const params = useParams();
+    const [inStock, setInStock] = useState(true);
+    const [itemDetails, setItemDetails] = useState({});
+
+    useEffect (() => {
+        axios.get(`http://localhost:8080/inventory/${params.id}`).then(res => {
+            setItemDetails(res.data[0]);
+        }).catch(err => {
+            console.log(err);
+        });
+
+        if(itemDetails.status === "In Stock" && itemDetails.quantity > 0) {
```

Comment:
```suggestion
        if(itemDetails?.status === "In Stock" && itemDetails?.quantity > 0) {
```

If visiting this route with an invalid inventory ID, the page crashes as it will attempt to get `status` on `itemDetails`, which is still undefined.